nailgun/entities.py

```diff
@@ -4005,9 +4006,12 @@ def update(self, fields=None):
 
     def update_payload(self, fields=None):
         """Wrap submitted data within an extra dict."""
-        return {
-            u'organization': super(Organization, self).update_payload(fields)
-        }
+        org_payload = super(Organization, self).update_payload(fields)
+        payload = {u'organization': org_payload}
+        if 'redhat_repository_url' in org_payload:
+            rh_repo_url = org_payload['redhat_repository_url']
```

Comment:
My concern was rather this:
update_payload() will generate smth like this:
```json
{
    "organization": {
        ...
        "redhat_repository_url": "http://example.com/"
    },
    "redhat_repository_url": "http://example.com"
}
```
The fact it's redundant is a different question, but what's about satellite? Will it normally process extra parameter which is most likely unexpected?